### ISSUE TITLE
pull image failed on Apple M1 Pro

### ISSUE THREAD

**Issue by glovebx** [odoo/docker#396]
execute command on Apple M1 Pro: 
    docker pull odoo

output error message:
    Using default tag: latest
    latest: Pulling from library/odoo
    no matching manifest for linux/arm64/v8 in the manifest list entries

**Comment by floflock** [comment]
There is an open issue for that: https://github.com/odoo/docker/issues/349

**Comment by The-macharia** [comment]
when pulling this image add the param --platform=linux/amd64

**Comment by floflock** [comment]
@The-macharia `linux/amd64` is not `linux/arm64` and the `linux/arm64` image is not available yet.

**Comment by The-macharia** [comment]
Yes, that's true. But I can successfully run the Odoo image on docker with mac M1, by explicitly specifying the platform.
